cmd/main.go

```diff
@@ -93,6 +92,21 @@ func parseConfig() (spirev1alpha1.ControllerManagerConfig, ctrl.Options, []*rege
 			"Command-line flags override configuration from this file.")
 	flag.StringVar(&spireAPISocketFlag, "spire-api-socket", "", "The path to the SPIRE API socket (deprecated; use the config file)")
 
+	// TODO: may we get metric, probeAdrr and leader?
```

Comment:
Has this commented code been addressed elsewhere? the features provided by these configurables are still necessary, especially leader election....